wasm.md

```diff
@@ -934,16 +939,16 @@ The `offset` parameter is added to the the address given on the stack, resulting
 The `align` parameter is for optimization only and is not allowed to influence the semantics, so we ignore it.
 
 ```k
-    syntax MemArg ::= Offset | Align | Offset Align
-    syntax Offset ::= "offset" "=" Int
-    syntax Align  ::= "align"  "=" Int
- // ----------------------------------
+    syntax MemArg ::= OffsetArg | AlignArg | OffsetArg AlignArg
+    syntax OffsetArg ::= "offset" "=" Int
+    syntax AlignArg  ::= "align"  "=" Int
```

Comment:
Maybe we can change it to "offset=" and "align=", because in the specs it says "no white space is allowed around the ‘=’".

Reply:
Oh, in that case sure. Might help the parser as well.